consensus/istanbul/core/handler.go

```diff
@@ -32,8 +32,23 @@ func (c *core) ParentCommits() MessageSet {
 
 // Start implements core.Engine.Start
 func (c *core) Start() error {
-	// Start a new round from last sequence + 1
-	c.startNewRound(common.Big0)
+	headBlock, headAuthor := c.backend.GetCurrentHeadBlockAndAuthor()
```

Comment:
Does it matter that we may not be synced by the time we call `Start()`?

Reply:
I haven't change the behavior.

Before we called `startNewRound(common.Big0)`. Now, instead, i copied all the significant lines that would have been executed on that precise scenario.

So, not sure how IBFT is supposed to work in this scenario.